### DISCUSSION TITLE
What versions are we running with?

### DISCUSSION THREAD

**Discussion by schmidtw** [xmidt-org/xmidt#68] [General]
As of this post, we're running with these versions:

- webpa/tr1d1um - `v0.7.4`
- xmidt/themis - `v0.4.8`
- xmidt/scytale - `v0.6.5`
- xmidt/petasos - `v0.1.7`
- xmidt/caduceus - `v0.6.6`
- xmidt/argus - `v0.9.0`
- xmidt/talaria - `v0.6.5`
- consul - `v1.12.2`

We tried using `consul 1.13.1` and it did not like our `consul 1.12.2` servers.  So we're going to keep using the `1.12.x` series from Consul for now.

**Comment by ctorrao** [comment]
Hello,

It's great to know that info from you, to have some insurance that we are in the right versions path.

Currently we are in a upgrade phase and testing the following versions:

- xmidt/tr1d1um:v0.7.4
- xmidt/themis:v0.4.7
- xmidt/scytale:v0.6.5
- xmidt/petasos:v0.1.7
- xmidt/caduceus:v0.6.6
- xmidt/argus:v0.9.0
- xmidt/talaria:v0.6.5
- consul:1.12.2
- yugabytedb/yugabyte:2.9.0.0-b4

@schmidtw, we are not using right now the themis v0.4.8 because it was missing in the dockerhub repo. I've open this issue: https://github.com/xmidt-org/themis/issues/103

**Reply by schmidtw** [reply]
I put a change into the pipeline in the hopes that it was simply an auth issue.  Once we get confirmation that it works we'll release everything fresh.
